Firebase/Core/FIRApp.m

```diff
@@ -159,8 +161,9 @@ + (void)configureWithName:(NSString *)name options:(FIROptions *)options {
 
   if ([name isEqualToString:kFIRDefaultAppName]) {
     if (sDefaultApp) {
```

Comment:
It looks like `sDefaultApp` is not protected from the race condition here, though it's protected with `@synchronized` in other places. Should we add `@synchronized(self)` here as well?

Reply:
There are a few other locations where it's not protected, so I think in this case it's probably best to leave it as is to avoid any extra latency and other issues. If we start seeing issues I'm fine to lock it down though! Does that sound like a good plan?

Reply:
Should be OK for now, but I feel we should bring consistency to the way we access the static variables in the class.